### DISCUSSION TITLE
Roadmap for Module Federation

### DISCUSSION THREAD

**Discussion by XiaofengXie16** [module-federation/module-federation-examples#642] [General]
Is there a roadmap for new features that will be added to module federation?

(I noticed that one of the new features would be allowing HMR updates bubble between builds which stated in the Webpack 5 roadmap. )

**Comment by jacob-ebey** [comment]
Not on my end. Federation is a stupid simple interface of "init" and "get" and I expect it to say that way. Anything else would be bug fixes / dev improvements like HMR.

**Comment by XiaofengXie16** [comment]
Thank you for your response.
I noticed that there are several items listed in the  project section of the webpack repo(https://github.com/webpack/webpack/projects/6) .I am curious if there are the improvements that you mentioned above.

**Reply by jacob-ebey** [reply]
😕 Not sure what you mean. You can track the webpack roadmap over at the webpack website: https://webpack.js.org/blog/2020-12-08-roadmap-2021/

**Comment by XiaofengXie16** [comment]
![image](https://user-images.githubusercontent.com/13548920/105532632-153daa00-5cb9-11eb-942e-029baaf1d0e3.png)
I am sorry I confused you. This is what I saw.

**Reply by jacob-ebey** [reply]
Yes, track things there. This is not a place for webpack planning or commitments. I personally do not have a say in what the webpack org priorities are.

**Comment by ScriptedAlchemy** [comment]
I'll look at outlining my initiatives here since I'm going to be drastically expanding the ecosystem
